index.js

```diff
@@ -2,16 +2,37 @@
   'use strict';
 
   var express = require('express');
+  var session = require('express-session');
   var bodyParser = require('body-parser');
   var compress = require('compression');
   var glob = require('glob');
+  var passport = require('passport');
+  var Sequelize = require('sequelize');
+  var SequelizeStore = require('connect-sequelize')(session);
+  var cookieParser = require('cookie-parser');
 
   var config = require('./config/config');
-
-  require('./database/start_database')();
+  var mysql_config = require('./config/mysql_config');
+  var session_config = require('./config/session_config');
 
   var app = express();
 
+  var db = new Sequelize(mysql_config.database, mysql_config.user, mysql_config.pass, 
+      { logging: false });
+  
+  //session store
+  app.use(cookieParser());
+  app.use(session(
+    { 
+      secret: session_config.secret,
+      resave: false, 
+      saveUninitialized: false,
+      store: new SequelizeStore(db, {}, 'user_sessions')
```

Comment:
I'll look into different ways to keep store this that potentially doesn't use an external package but that's out of the scope of this PR